docs/run-keep-ecdsa.adoc

```diff
@@ -260,6 +282,47 @@ reference].
 |No
 |===
 
+[%header,cols=4*]
+|===
+|`Extensions.TBTC`
+|Description
+|Default
+|Required
+
+|`LiquidationRecoveryTimeout`
+|The amount of time your client will try to communicate with the other signers to recover the underlying bitcoin from a deposit after it has been liquidated.
+|"48h"
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`Extensions.TBTC.Bitcoin`
+|Description
+|Default
+|Required
+
+|`BeneficiaryAddress`
+|The btc address or *pub (xpub, ypub, zpub) that you would like recovered btc funds to be sent too.
```

Comment:
What do you think about adding some sample values? Especially for *pub so a user knows how should they format the value, e.g. that `xpub` prefix is expected in the provided value. Maybe this will be too much data to display the table properly so we could add a separate section in the document for liquidation recovery with a brief description?

Reply:
I like that idea - mind if I do it in a follow-up PR?

Reply:
Yes, please!